_includes/header.html

```diff
@@ -1,17 +1,17 @@
-<a class="content-skip" href="#content">Skip to content</a>
+    <a id="content-skip" href="#content" tabindex="1">Skip to content</a>
 <header>
     <script src="/js/header.js"></script>
-    <a href="/"><img id="logo" src="/assets/img/fist.svg"></a>
+    <a href="/"><img id="logo" src="/assets/img/fist.svg" alt="A geometric red fist clenching a wrench."></a>
     <div class="header-group">
         <h2 class="title"><a href="/">Vae Victis</a></h2>
         <h5 class="title-simple"><a href="/">1418</a></h2>
         <h3><a href="/">FRC Team 1418 - George Mason High School</a></h3>
     </div>
     <ul class="social">
-        <li><a href="mailto:robotics1418ATSYMBOLgmailPERIODcom"><img src="/assets/img/social/email.svg"></a></li>
-        <li><a href="http://facebook.com/robotics1418"><img src="/assets/img/social/facebook.svg"></a></li>
-        <li><a href="http://github.com/frc1418"><img src="/assets/img/social/github.svg"></a></li>
-        <li><a href="http://twitter.com/Robotics1418"><img src="/assets/img/social/twitter.svg"></a></li>
+        <li><a href="mailto:robotics1418ATSYMBOLgmailPERIODcom"><img src="/assets/img/social/email.svg" alt="A white email icon."></a></li>
```

Comment:
White isn't necessary

Reply:
I'd say something like "Email icon" and "Facebook logo" for these. No periods necessary, by the way.